### ISSUE TITLE
BUG? Some settings not exported (that should), some exported that should not

### ISSUE THREAD

**Issue by smayer97** [proginosko/LeechBlockNG#275]
Not sure if this is by design or a bug but I noticed that not ALL settings are exported. And for some settings, it may be a good idea to not export them.

Settings I know that are not exported that I thing should be are:
- Prevent access to about:addons
- Prevent access to about:support

OR if they are exported (from Firefox) do not get applied to their equivalent when imported into Chrome, which the closest I can see would be:
- Prevent access to Extensions
- Prevent access to Settings

Some settings that are exported that I think should not be are:
- Control Access password
Strangely, the Temporary Override Password IS exported.

I would recommend at least these changes/corrections. BUT there may be others too.

**Comment by proginosko** [comment]
Thanks for the report.

I can see that it makes some sense to treat "Prevent access to about:addons" (Firefox) as equivalent to "Prevent access to Extensions page" (Chrome). But I don't think that's the case for about:support (Firefox) and Settings (Chrome). The latter options are quite different, and treating them as interchangeable would be counterintuitive and potentially dangerous. So I'll add the translation in the first case, but not the second.

**Comment by smayer97** [comment]
Thanks. That would be nice. Are there others that map like this? 

What about the passwords? There is an inconsistency of at least one getting exported but not the other(s).

**Comment by proginosko** [comment]
The access control password is exported (I just tested it). Perhaps take a second look at your exported file?

**Comment by smayer97** [comment]
Sorry, you are correct. I can confirm that both passwords are getting exported. 

I never had issue with the Access Control Password not exporting but rather it seemed like the Temporary Override Password was not because the export file uses a cryptic field name label so it was not obvious to find.

So, this part is now obviously not a bug. Instead, I would recommend turning this part into a feature request:
- provide the option to export or not the passwords in the export file. 
The reason is that since it is a simple text file, it may be desired to not allow that info to be easily found by other eyes sharing the computer but one may still want to create an easily accessible backup of the other settings.

Also, Is there anymore other mapping of settings that need to be done between Firefox and Chrome? I did not go through all of them... I only identified the one or two that I stumbled across.
